goc/install/install-oasis

```diff
@@ -191,7 +189,6 @@ for REPO in oasis config-osg; do
 	CVMFS_AUFS_WARNING=false cvmfs_server mkfs -m -o oasis $REPO.opensciencegrid.org
     fi
     echo "CVMFS_AUFS_WARNING=false" >>/etc/cvmfs/repositories.d/$REPO.opensciencegrid.org/server.conf
-    echo "CVMFS_IGNORE_XDIR_HARDLINKS=true" >>/etc/cvmfs/repositories.d/$REPO.opensciencegrid.org/server.conf
```

Comment:
It is irrelevant with overlayfs; overlayfs does not support real hardlinks at all, it makes copies of hardlinked files.